backend/src/main/java/se/chalmers/dat265/group1/model/MagicsData.java

```diff
@@ -0,0 +1,23 @@
+package se.chalmers.dat265.group1.model;
+
+public class MagicsData extends DataModel {
+    private int id;
+    private int digitalPrintID;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "digitalPrintID" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)